test/e2e/create_enabled_hibernate_reconcile_delete_test.go

```diff
@@ -36,6 +36,9 @@ var _ = Describe("Shoot Rsyslog Relp Extension Tests", func() {
 		By("Install rsyslog-relp unit on Shoot nodes")
 		Expect(execInShootNode(ctx, f.ShootFramework.SeedClient, f.Logger, f.ShootFramework.ShootSeedNamespace(), "apt-get update && apt-get install -y rsyslog-relp")).To(Succeed())
 
+		By("Wait for rsyslog to get configured on Shoot node")
+		time.Sleep(15 * time.Second)
```

Comment:
You could use `Eventually` and put the verifications in that `Eventually` with the given timeout (15s) instead of sleeping.

Reply:
The verification already use an `Eventually` or `Consistently` blocks depending on what they check , perhaps the better solution here would have been to simply increase the timeouts of those blocks to 15s. I'll try it out.

Reply:
Yep, sorry, that I missed this. Sounds better to increase the corresponding `Eventually` and `Consistently` timeouts. 👍